BUILDING.md

```diff
@@ -35,26 +35,33 @@ Note: on Windows invoke the `gfsh.bat` script to print the version string.
 
 The following steps have been tested with:
 
-* **IntelliJ IDEA 2018.2.2**
+* **IntelliJ IDEA 2018.3.5**
 
-1. Run `./gradlew --parallel devBuild` from Geode repository root to create compiler generated source
+1. Run `./gradlew --parallel generate` from Geode repository root to create compiler generated source.
+   1. Alternatively (and to ensure these sources stay up-to-date):
```

Comment:
Is it worth having the alternative listed here or just putting one opinionated way?  When following this on a fresh clone I didn't like skipping to step 2 to come back to step 1 (but I'm bad at following directions, so likely a me problem).